Add WRLinux product WRLinux8 and WRLinux1019 support (V2)

dehuo0 opened the pull request:
Description:
Rename wrlinux to wrlinux8 to specify the Wind River Linux product version number.
Since there are serval version Wind River products start to support SCAP e.g. WRLinux8,
WRLinux1019, the supported profiles and functions would be different between these
version products. The old source code for wrlinux in SCAP is just for WRLinux8, so rename
the wrlinux to wrlinux8.
Add the WRLinux1019 platform support, provide the relevant functional files for the WRLinux1019
e.g. profiles, cpe directory etc.

Rationale:
Add home directories for WRLinux8 and WRLinux1019;
Add multi_platform_wrlinux platform to shared.xml files for wrlinux supported rules;
Add wrlinux prodtype to rule.yml files for wrlinux supported rules;
Add multi_platform_wrlinux platform to remediate bash scripts for wrlinux supported rules;

The update in this version pull request is as below:
1. Update the description in CMakeLists.txt to identify WRLinux8 and WRLinux1019 not only WRLinux;
2. Update the installed_OS_is_wrlinux8.xml and installed_OS_is_wrlinux1019.xml to add more version check;

cipherboy commented:
@opensca-ci test this please

yuumasato commented:
@openscap-ci add to white list

yuumasato commented:
@dehuo0 Thanks for the updated. There was no need for new PR, just pushing the commits to the one already open would suffice.

The changes look good to me.
Now, there are some conflicts, I apologize for this hassle. Just rebasing your branch on top of latest master should resolve them.
Would you like help with that?

dehuo0 commented:
> 
> 
> @dehuo0 Thanks for the updated. There was no need for new PR, just pushing the commits to the one already open would suffice.
> 
> The changes look good to me.
> Now, there are some conflicts, I apologize for this hassle. Just rebasing your branch on top of latest master should resolve them.
> Would you like help with that?

Thank you. I've rebased to the master and updated my patches, please help to check.

jan-cerny commented on wrlinux8/templates/Makefile:
```diff
@@ -0,0 +1,15 @@
+OUTPUT:=output
+OUTPUT_BASH:=$(OUTPUT)/bash
+CSV_DIR:=csv
+
```
I wonder what is the Makefile good for. We don't have makefiles anywehere else than in WRLinux directories. We use CMake for a long time.

jan-cerny replied:
I think the Makefiles can be removed, I can't find any place where they're called.

dehuo0 replied:
Thanks for the reminding. I will remove the Makefile from WRLinux directories.

jan-cerny commented on CMakeLists.txt:
```diff
@@ -241,7 +242,8 @@ message(STATUS "SUSE 12: ${SSG_PRODUCT_SLE12}")
 message(STATUS "Ubuntu 18.04: ${SSG_PRODUCT_UBUNTU1804}")
-message(STATUS "WRLinux: ${SSG_PRODUCT_WRLINUX}")
+message(STATUS "WRLinux 8: ${SSG_PRODUCT_WRLINUX8}")
+message(STATUS "WRLinux 1019: ${SSG_PRODUCT_WRLINUX1019}")
```
Needs to be added in `/build_product` script around line 74.

dehuo0 replied:
Thank you. I will supply these to build_product script.

yuumasato replied:
@jan-cerny Nice catch!

jan-cerny commented:
@openscap-ci test this please

jan-cerny commented:
Thank you for your great contribution!

dehuo0 commented:
Why there are so many rules no oval check in RedHat7 profile, for example audit_rules_privileged_commands_*
